### PR TITLE
feat(api): remove the range slider implementation

### PR DESCRIPTION
**Pull request by mthuret**

### CODE REVIEW

File: packages/react-instantsearch/src/widgets/Range.js [hunk 1]
```diff
@@ -1,4 +1,66 @@
...
+          </li>
+        </ol>
+      </div>
+    );
+  }
+}
+
+ const AirbnbRheostatConnected = connectRange(AirbnbRheostat);
+ */
+export default connectRange(null);
```
Comment (vvo) [inline packages/react-instantsearch/src/widgets/Range.js]: Should we have a console.error() that says "We do not provide a Slider, see documentation for how to do it easily" something like that?

Comment (mthuret) [inline packages/react-instantsearch/src/widgets/Range.js]: yes good idea

Comment (mthuret) [inline packages/react-instantsearch/src/widgets/Range.js]: also, I removed the widget from the namespace.

**Comment by vvo** [comment]
LGTM

**Pull request merged**